src/HomeNet/HomeNet.conf

```diff
@@ -102,15 +102,39 @@ client_customer_interface_port = 16988
 #
 # To generate PFX certificate, you can use OpenSSL:
 #
-#   openssl req -x509 -newkey rsa:4096 -keyout HomeNet.key -out HomeNet.cer -days 36500
+#   openssl req -x509 -newkey rsa:4096 -keyout HomeNet.key -out HomeNet.cer -days 365000

```

Comment:
It is self signed invalid certificate, why would you replace that? I have not tested it with expired certificates, but it is likely that it will work anyway, so even if we set expiration to 1 day, it might just work and no one will replace the cert anyway. 

If we start doing some validations of certificates, then it might start matter. Until then, it is probably a non-issue.